packages/browser/src/node/providers/none.ts

```diff
@@ -0,0 +1,40 @@
+import type { Awaitable } from 'vitest'
+import type { BrowserProvider, WorkspaceProject } from 'vitest/node'
+
+export class NoneBrowserProvider implements BrowserProvider {
+  public name = 'none'
+  private ctx!: WorkspaceProject
+  private open = false
+
+  getSupportedBrowsers() {
+    // `none` supports any browser
+    return []
```

Comment:
The comment may sound confusing, because [] and any browser may not be aligned. It's just a suggestion, you can keep it as is or write more descriptions.

```
// `none` is not restricted to certain browsers.
```